packages/explat-client-react-helpers/src/index.tsx

```diff
@@ -1,8 +1,9 @@
 import { useState, useEffect } from 'react';
 import * as React from 'react';
 import type { ExPlatClient, ExperimentAssignment } from '@automattic/explat-client';
+export { ExPlatClient, ExperimentAssignment };
```

Comment:
This didn't seem necessary. Are you sure it was?

Reply:
Hey, thanks for looking into this! 

Without this re-export I had to add this line: https://github.com/woocommerce/woocommerce-admin/blob/08bf75461ac8ba6754bc95f261c0fd6cf375068a/packages/explat/tsconfig.json#L7 or there would be an error on build

Reply:
In WC Admin I did the following:

1. Apply the patch below disabling patch package and the ts alias you mention. 
1. `rm -fr node_modules; pnpm i` (install clean node modules)
1. Manually edit _this package_ where installed in node_modules. **Important note** you'll need to edit the declaration file at `node_modules/@automattic/explat-client-react-helpers/dist/types/index.d.ts`! I only added exports to the two interfaces defined there, I did not add any re-exports. 
1. `cd packages/explat`
1. `pnpm run clean && pnpm run build`

Seems to workout the re-export.

```diff
diff --git a/package.json b/package.json
index 4e2f5fd8e..988bff090 100644
--- a/package.json
+++ b/package.json
@@ -71,7 +71,6 @@
 		"test:help": "wp-scripts test-unit-js --help",
 		"test:php": "docker-compose -f docker/wc-admin-php-test-suite/docker-compose.yml run --rm phpunit",
 		"posttest:php": "docker-compose -f docker/wc-admin-php-test-suite/docker-compose.yml down",
-		"postinstall": "patch-package",
 		"test:update-snapshots": "pnpm run test:client -- --updateSnapshot && pnpm run --filter @woocommerce/components test:update-snapshots",
 		"test:watch": "tsc --build || concurrently \"pnpm run test:client -- --watch\" \"pnpm run:packages -- test:nobuild --parallel -- --watch\"",
 		"test:zip": "pnpm run clean && composer i && ./bin/build-test-zip.sh",
diff --git a/packages/explat/tsconfig.json b/packages/explat/tsconfig.json
index 9f49af1cd..6ac6ac42d 100644
--- a/packages/explat/tsconfig.json
+++ b/packages/explat/tsconfig.json
@@ -2,9 +2,6 @@
 	"extends": "../tsconfig",
 	"compilerOptions": {
 		"rootDir": "src",
-		"outDir": "build-module",
-		"paths": {
-			"@automattic/explat-client": [ "../../node_modules/@automattic/explat-client/src" ],
-		}
-	},
-}
\ No newline at end of file
+		"outDir": "build-module"
+	}
+}
```

Reply:
Thanks @sirreal, you're right - I can't reproduce it anymore.
Might have been a npm missing dependency issue or something that was resolved when we moved to pnpm just this week.

Appreciate your help!